ci/travis/install-bazel.sh

```diff
@@ -21,3 +21,9 @@ wget -O install.sh $URL
 chmod +x install.sh
 ./install.sh --user
 rm -f install.sh
+
+if [[ "$TRAVIS" == "TRAVIS"  ]]; then
+  # Use bazel disk cache if this script is running in Travis.
+  mkdir -p $HOME/ray-bazel-cache
+  echo "build --disk_cache=$HOME/ray-bazel-cache" >> $HOME/.bazelrc
```

Comment:
I think this need to be $RAY_ROOT/.bazelrc for it to be effective. 

```
FILE_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
RAY_ROOT = "$FILE_DIR/../.."
```

Reply:
Bazel also reads `.bazelrc` file in the home directory. 
I'm rerunning the CI jobs to see if time will be shorten.

Reply:
It seems that Travis doesn't allow modifying files in the home dir.

Reply:
Verified that this can work. (I thought it didn't work because I used wrong env var.)
![image](https://user-images.githubusercontent.com/2883335/60871442-556b4200-a265-11e9-9e25-6812fc78c0d0.png)